analyze.py

```diff
@@ -100,7 +126,20 @@ def http_analyze(self, responses):
             table.header(['Username', 'Password', 'Resp Code' , 'Resp Length'])
             for x in len_indicies:
                 table.add_row([responses[x][0], responses[x][1], responses[x][2], responses[x][3]])
+            
+            # Calling notifications if specified
+            if self.notify == 'slack':
+                slack(self.webhook, self.host)
+            elif self.notify == 'teams':
+                teams(self.webhook, self.host)
+            elif self.notify == 'discord':
+                teams(self.webhook, self.host)
+
             print(table.draw())
+
+            # Returning true to indicate a successfully guessed credential
+            return True
+
         else:
             self.colors.color_print('[-] No outliers found or not enough data to find statistical significance', self.colors.red)
 
```

Comment:
return false for the `check_sleep()` call in `spraycharles.py`

```suggestion
            return False
```